src/main/java/com/github/jcustenborder/kafka/connect/spooldir/AbstractCleanUpPolicy.java

```diff
@@ -110,6 +130,25 @@ public void success() throws IOException {
     }
   }
 
+  static class MoveByDate extends AbstractCleanUpPolicy {
```

Comment:
File should probably not be named AbstractCleanUpPolicy.java if its going to contain all of the cleanup policies.